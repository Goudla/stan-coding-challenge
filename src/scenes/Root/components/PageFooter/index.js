import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class PageFooter extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="page-footer">
        {children}
      </div>
    );
  }
}

export default PageFooter;
