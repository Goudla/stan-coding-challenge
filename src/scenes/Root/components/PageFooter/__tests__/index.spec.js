import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import PageFooter from '../';

describe('<PageFooter />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<PageFooter>Test</PageFooter>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
