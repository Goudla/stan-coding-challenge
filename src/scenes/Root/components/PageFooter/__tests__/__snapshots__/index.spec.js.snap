// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageFooter /> Should render 1`] = `
<div
  className="page-footer"
>
  Test
</div>
`;
